pyabc/weighted_statistics.py

```diff
@@ -29,7 +29,7 @@ def weighted_quantile(points, weights=None, alpha=0.5):
         weights = weights[sorted_indices]
 
     cs = sp.cumsum(weights)
-    quantile = sp.interp(alpha, cs - (1-alpha)*weights, points)
+    quantile = sp.interp(alpha, cs - 0.5*weights, points)
```

Comment:
why `0.5` ?

Reply:
good point. I had changed that when I went from weighted median to arbitrary weighted quantiles. The point is to subtract a bit in order to get correct quantiles, but after revising, I think that 0.5 is actually the right value for arbitrary quantiles. See also the discussion in https://stackoverflow.com/questions/21844024/weighted-percentile-using-numpy.

Why do we subtract something? Imagine points [a,b,c] with a<b<c and you want a 33% Percentile (=0.33 quantile). Then you would want a point between a and b (s.t. 1/3 are to the left and 2/3 to the right). If we do not subtract, we would get the point a, but by subtracting 0.5, the weight at a exactly is lower s.t. we get a point between a and b, as intended.